.travis/build-deploy.sh

```diff
@@ -27,8 +27,21 @@ TRAVIS_COMMIT_RANGE=${TRAVIS_COMMIT_RANGE}"
     fi
 }
 
+function upload_baseline() {
```

Comment:
We'll probably need to execute the commands here in a subshell and disable fast-fail, too.
Otherwise, the whole script will exit when the first command fails (e.g. rsync uploading to sourceforge) and we won't be able to react on `$?` exit codes.

Have a look at line 74 https://github.com/pmd/pmd/pull/1250/files#diff-999b8b90ab0a1755791c28e780bd4ef2R74 to see an example.

Reply:
Ah, I see, it's now inside the subshell together with the other calls to sourceforge. :+1: